frontend/src/app/drive/layout.tsx

```diff
@@ -23,7 +23,7 @@ export default function AppLayout({
 }: Readonly<{
   children: React.ReactNode;
 }>) {
-  const { subscription } = useUserStore();
+  const subscription = useUserStore(({ subscription }) => subscription);
```

Comment:
```suggestion
  const subscription = useUserStore((state) => state.subscription);
```
https://zustand.docs.pmnd.rs/getting-started/introduction